packages/squiggle-lang/src/fr/list.ts

```diff
@@ -97,14 +96,14 @@ export const library = [
     definitions: [
       makeDefinition([frArray(frAny), frLambda], ([array, lambda], context) => {
         const mapped: Value[] = new Array(array.length);
-        const counts = lambda.paramCounts();
+        const counts = lambda.parameterCounts();
 
         // this code is intentionally duplicated for performance reasons
-        if (includes(counts, 2)) {
+        if (counts.includes(2)) {
```

Comment:
You haven't implemented my suggestion in https://github.com/quantified-uncertainty/squiggle/pull/2334#discussion_r1357239146 to throw on ambiguity, did you miss it or do you disagree?

Reply:
Sorry - I think I missed that one in the changes. That said, we will be removing much of this in this PR:
https://github.com/quantified-uncertainty/squiggle/pull/2290

(That said, that PR does this by having frLambda(1) and frLambda(2) - it doesn't detect ambiguity, which might be bad)